internal/k8shandler/forwarding_test.go

```diff
@@ -623,8 +623,7 @@ var _ = Describe("Normalizing forwarder", func() {
 					_, _ = request.generateCollectorConfig()
 
 					Expect(len(request.ForwarderSpec.Outputs) == 1).To(BeTrue())
-					Expect(request.ForwarderSpec.Outputs[0].Elasticsearch.StructuredTypeKey).To(Equal("kubernetes.labels.mylabel"))
```

Comment:
can we enforce this at the API level that outputDefaults can be used only if there is an output named `default` ? if not, can we update this in status that "`outputDefaults` is redundant because there is no output named `default`" ?
otherwise how does user get any feedback ?